README.md

```diff
@@ -1,8 +1,59 @@
-# vscode-aks-tools
+# Azure Kubernetes Service (AKS) Extension for Visual Studio Code (Preview)
 
-* View your AKS clusters in the Kubernetes extension cloud explorer
-* Add AKS clusters to kubeconfig via the GUI
-* Run AppLens detector diagnostics on your AKS clusters
+The Visual Studio Code extension for Azure Kubernetes Service helps you view your AKS cluster, merge and save kubeconfig, perform AKS Diagnostics checks, and run AKS Periscope within VS Code.
+
+## Prerequisites
+
+Ensure that you have installed the [Kubernetes Tools](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools) extension for VS Code prior to installing the Azure Kubernetes Service Extension for Visual Studio Code following these [instructions](https://code.visualstudio.com/docs/azure/kubernetes#_install-the-kubernetes-extension).
+
+## Installation
+
+1. Download and install the [Azure Kubernetes Service extension](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-aks-tools) for Visual Studio Code.
+
+2. Wait for the extension to finish installing then reload Visual Studio Code when prompted.
+
+3. Once the installation is complete, you'll see a section named **Azure** under **Clouds**.
+
+4. Sign in to your Azure Account by clicking **Sign in to Azure…**
+
+## Features
+
+### Merge and Save Kubeconfig
+
+Log-in with your Azure credentials and view all AKS clusters in your Azure subscription(s) within VS Code. Right click on your AKS cluster and click on **Merge into Kubeconfig** to add the selected AKS cluster to the kubeconfig file or click on **Save Kubeconfig** to save the kubeconfig of the selected AKS cluster.
+
+### AKS Diagnostics
+
+Right click on your AKS cluster and click on **Run AKS Diagnostics** to run a series of checks on your AKS cluster's backend telemetry to get diagnostics information on your network configuration and recommended solutions for network and connectivity issues. Also, to perform more checks on your AKS cluster to troubleshoot and get recommended solutions, click on the AKS Diagnostics link at the top of the page to open it for the selected cluster. For more information on AKS Diagnostics, visit [AKS Diagnostics Overview](https://docs.microsoft.com/azure/aks/concepts-diagnostics).
+
+### AKS Periscope
+
+Right click on your AKS cluster and click on **Run AKS Periscope** to collect node and pod logs for your AKS cluster and to export them to an Azure storage account. Upon selecting the option, a web view will load providing you the option to generate a downloadable link for the collected logs as well as a shareable link with 7-day expiry.
```

Comment:
:bulb: What do you guys think about:  Probable emphasis on `downloadable link` and `shareable link` ? i.e. either with screenshot or make it **bold**, for clarity.

Reply:
This could be taken care of with a screenshot of the web view. Would you be able to add a screenshot from your local?

Reply:
Cool, Sounds good. sent you the screenshot, essentially since PR is from your Fork, I can either send a PR to your fork and them merge it or you can add that screenshot and add it to the placeholder.

Reply:
Here you go: to help I have added PR for your Fork:

* https://github.com/yunjchoi/vscode-aks-tools/pull/2 

Added missing screenshots, please feel free to review and merge in your fork, that way it will appear I this PR. Hope this helps, thanks. :)